sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/parquet/ParquetWrite.scala

```diff
@@ -72,6 +72,8 @@ case class ParquetWrite(
 
     ParquetOutputFormat.setWriteSupportClass(job, classOf[ParquetWriteSupport])
 
+    // TODO: Check V2 insert schema together.
```

Comment:
it's quite weird if we only fix parquet v2.

Reply:
Can we revert it? file source v2 is not enabled by default anyway and we don't need to rush for a partial fix.

Reply:
Done